modules/ROOT/pages/index.adoc

```diff
@@ -3,15 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Security provides a layered approach to secure your application network. These layers work together to protect both the application network and the network's individual nodes by controlling access to APIs, enforcing policies, and proxying all inbound or outbound traffic to mitigate external threats and attacks. +
-Anypoint Security provides you with a dedicated endpoint to detect attacks and validate traffic without taxing your network implementations.
+Anypoint Security provides a layered approach to secure your application network. These layers work together to protect both the application network and the network's individual nodes by controlling access to APIs, enforcing policies, and proxying all inbound or outbound traffic to mitigate external threats and attacks. 
```

Comment:
There is no line break or new <p> at all now--this is all one para. If you want a line break, you need one more line. If you meant this to be just one para, disregard and carry on. :-)

Reply:
Yah, I think the plus sign was left in by accident.